CHANGELOG.md

```diff
@@ -5,17 +5,27 @@
 **This release is rolling out to all regions**
 
 ** Service Updates**
-* As announced, the migration to VMSS/SLB AKS clusters by default is under way,
+* As announced the migration to VMSS/SLB AKS clusters by default is under way,
   if you are using the `aks-preview` Azure CLI extension, all clusters created
   are now defaulted to VMSS/SLB.
 * AKS Kubernetes 1.10 support will end-of-lifed on Oct 25, 2019
 * AKS Kubernetes 1.11 & 1.12 support will end-of-lifed on Dec 9, 2019
 * New Documentation additions:
   * [Authenticate with Azure Container Registry from AKS](https://docs.microsoft.com/en-us/azure/aks/cluster-container-registry-integration)
   * [Security hardening in AKS virtual machine hosts](https://docs.microsoft.com/en-us/azure/aks/security-hardened-vm-host-image)
+* The AKS team is pleased to announce the new `aks-periscope` tool.
+  * `aks-periscope` is a tool deployed via a Kubernetes Daemonset for customers
+    to use in order to diagnose their cluster issues
+    that are caused by incorrect configuration of the cluster/environment
+    (such as networking or permission issues). This tool will allows AKS
+    customers to run initial diagnostics and collect logs that helps them
```

Comment:
Just being a bit picky (as I'm also polishing up the actual GitHub too), "AKS Periscope will allow AKS customers to run initial diagnostics and collect logs to help them analyze and identify the underlying problems."